docs/basics/desktop-app-basics.md

```diff
@@ -1,6 +1,101 @@
+
+The installation guides linked above are straight forward and easy to follow; simply follow the instructions that correspond to your operating system and you will have IPFS Desktop going in just a few minutes.
+
+
+---
+
+
+## Add a file using IPFS Desktop
+
+Now that you have IPFS Desktop up and running, you can add the files that you wish to host on IPFS.
+
+1. First, open IPFS Desktop and make sure you are connected to IPFS. Navigate to the status screen by clicking on the `Status` tab on the left side of the app. At the top of the status screen you will see that it says "Connected to IPFS".
+
+1. Now that you know you are connected to IPFS, navigate to the files screen. To do this simply click on the `Files` tab on the left side of the app.
+
+1. On the files screen, you will see a button that says `+Import`, click on that and then select either `File` or `Folder`. If you are uploading just a single file to IPFS, select the `File` option. If you are uploading a folder containing multiple files, select the `Folder` option. Selecting either option will bring up a file selection window.
+
+1. In the file selection window, navigate to the file or folder that you wish to upload and host on IPFS. Once you have found the file or folder, simply double click on it.
+
+1. Your file has now been uploaded to IPFS!
+
+When files are uploaded to IPFS, they are automatically given a content identifier (CID). The CID for each file is listed directly under the file name on the `Files` page. To share a file that you are hosting on IPFS, simply click the 3 dots to the right of the desired file, select `Copy CID` and send the CID to who ever you would like to have access to the file.
+
+
+---
+
+
+## Download a file using IPFS Desktop
+
+Importing and downloading a file to your local storage using IPFS is an easy task with IPFS Desktop, these simple steps will walk you through the process.
+
+1. First, you will need the CID of the file that you wish to import and download. The CID needs to be shared with you, either by the owner of the file or by a peer who has previously imported the file to their IPFS node.
+
+1. In IPFS Desktop, navigate to the files screen.
+
+1. In the files screen, there will be a large button that says `+Import`, click on that.
+
+1. In the drop down menu that appears, select `From IPFS`, a new window will appear.
+
+1. In this new "Import from IPFS" window, paste the desired CID into the "Path or CID (required)" field. A couple of example CIDs are given for you to try it out. If you wish to give the file a specific name, there is also an optional field to do that.
+
+1. Once you have the desired CID and name filled out, click `Import`. The file will be automatically retrieved over IPFS to your IPFS node. The file will now appear in the list of files with the name that you have chosen.
```

Comment:
File is not retrieved, only the root block with  metadata like file size.
```suggestion
1. Once you have the desired CID and name filled out, click `Import`. The file size metadata will be automatically retrieved over IPFS, and it will now appear in the list of files with the name that you have chosen.
```